CampaignConfig/mdc2020_corsika.cfg

```diff
@@ -50,69 +51,69 @@ job_setup.postscript_9 = loggedMu2e.sh -c `cat torun.txt`
 job_setup.multifile = False
 job_setup.setup_local = True
 
-[stage_generation_cat_fcl]
-global.stage_name = CosmicDSStopsCORSIKACat
+[stage_resampler_fcl]
+global.stage_name = CosmicCORSIKA
 job_output.dest = %(outdir_fcl)s/%(stage_name)s/%(desc)s/fcl/
 
 job_setup.prescript = samweb list-file-locations --defname=sim.%(submitter)s.CosmicDSStopsCORSIKA.%(desc)s.art --schema=root | cut -f1 > inputs.txt
-job_setup.prescript_2 = echo '#include "Production/JobConfig/common/artcat.fcl"' >> template.fcl
-job_setup.prescript_3 = echo 'outputs.out.fileName: "sim.DSOWNER.CosmicDSStopsCORSIKACat.DSCONF.SEQ.art"' >> template.fcl
 executable.name = true
 executable_2.name = generate_fcl
-executable_2.arg_4 = --merge=50
-executable_2.arg_5 = --inputs=inputs.txt
-executable_2.arg_6 = --embed
-executable_2.arg_7 = template.fcl
-
+executable_2.arg_4 = --aux=1:physics.filters.cosmicResample.fileNames:inputs.txt
+executable_2.arg_5 = --run-number=1205
+executable_2.arg_6 = --events-per-job=200000
```

Comment:
This is OK for now, but in future there should be a script to compute it from the sam data